web/turkey/docs/build/html/_static/pygments.css

```diff
@@ -0,0 +1,65 @@
+.highlight .hll { background-color: #ffffcc }
```

Comment:
So it looks like I should have run `make gh-pages` and then opened a PR on `gh-pages`?